### ISSUE TITLE
macOS Installer Broken

### ISSUE THREAD

**Issue by MarcusHurney** [mymonero/mymonero-app-js#290]
When I download the macOS MyMonero desktop app, I get this error each time:
 
`“MyMonero” can’t be opened because Apple cannot check it for malicious software. This software needs to be updated. Contact the developer for more information.`

I'm running macOS Catalina v10.15.2.

Can you please look into this?

**Comment by MarcusHurney** [comment]
![Screen Shot 2020-01-09 at 8 25 45 PM](https://user-images.githubusercontent.com/13347886/72118147-405b9280-331e-11ea-8eb6-015044f0d7a4.png)

**Comment by cisarpan** [comment]
I'm having the same problem on MacOS Mojave. Anyone to help with this issue? 
Thank you very much.

**Comment by devinpearson** [comment]
We are in the process of sorting out the notarization to sop users getting this message.

However, you can work around it by clicking "OK" on in popup box and going to your System Preferences / Security & Privacy, and under the General tab you will see and open to open anyway and that will open the app.

**Comment by cisarpan** [comment]
@devinpearson  thank you for the help. The funny thing is in my System Preferences I have allowed apps be downloaded from App Store and identified developers. Usually there is a pop up with "open anyway" while installing new apps from other developers, but in this case no pop up. Anyway I tried numerous times and once after the first pop up, pictured above by MarcusHurney, second pop up appeared with options to click OK or Cancel, and that did the trick.
